### PR TITLE
Added Structor and Sealer classes with tests. Some refactoring cleanup. 

### PR DESCRIPTION
**Pull request by SmithSamuelM**

### CODE REVIEW

File: src/keri/core/counting.py [hunk 1]
```diff
@@ -165,18 +114,18 @@ class CounterCodex_2_0(MapCodex):
     BigTransReceiptQuadruples: str = '-0M'  # Big Trans Receipt Quadruple(s), pre+snu+dig+sig.
     FirstSeenReplayCouples: str = '-N'  # First Seen Replay Couple(s), fnu+dts.
     BigFirstSeenReplayCouples: str = '-0N'  # First Seen Replay Couple(s), fnu+dts.
-    TransIdxSigGroups: str = '-O'  # Trans Indexed Signature Group(s), pre+snu+dig+ControllerIdxSigs of qb64.
-    TransIdxSigGroups: str = '-0O'  # Big Trans Indexed Signature Group(s), pre+snu+dig+ControllerIdxSigs of qb64.
-    TransLastIdxSigGroups: str = '-P'  # Trans Last Est Evt Indexed Signature Group(s), pre+ControllerIdxSigs of qb64.
-    BigTransLastIdxSigGroups: str = '-0P'  # Big Trans Last Est Evt Indexed Signature Group(s), pre+ControllerIdxSigs of qb64.
+    TransIdxSigGroups: str = '-O'  # Trans Indexed Signature Group(s), pre+snu+dig+CtrControllerIdxSigs of qb64.
+    TransIdxSigGroups: str = '-0O'  # Big Trans Indexed Signature Group(s), pre+snu+dig+CtrControllerIdxSigs of qb64.
```
Comment (kentbull) [inline src/keri/core/counting.py:118]: looks like this got defined twice

Comment (SmithSamuelM) [reply src/keri/core/counting.py:118]: Nope one includes Last the other doesn't

Comment (kentbull) [reply src/keri/core/counting.py:118]: I was referring to lines 117 and 118. My IDE gave me a warning on `TransIdxSigGroups` being redefined

Comment (SmithSamuelM) [reply src/keri/core/counting.py:118]: Yes that is a typo. Second one should should start with Big. I will fix

**Comment by SmithSamuelM** [comment]
fixed with #744

**Pull request merged**